notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb

```diff
@@ -92,7 +92,7 @@
     {
```

Comment:
<p>Should this number in the statement be changed to <code>&gt;98%</code> or <code>~100%</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
@chauhankaranraj , I changed the percent value to 100%, as seen in the output of the code.